packages/analytics-js/src/services/StoreManager/StoreManager.ts

```diff
@@ -85,11 +95,10 @@ class StoreManager implements IStoreManager {
         }
         break;
       case MEMORY_STORAGE:
-        break;
       case NO_STORAGE:
         break;
       default:
```

Comment:
This case should be combined with cookie storage case.

Reply:
Addressed